Javascript/translator.test.js

```diff
@@ -0,0 +1,10 @@
+import  morseToText from "./translator.js";
```

Comment:
translator.js does not provide a default export 
```js
import {morseToText} from "./translator.js"
```

Reply:
Got it. Completed the test cases